$NetBSD: patch-ab,v 1.7 1999/11/24 17:34:10 hubertf Exp $

Place(s) where this was (most probably...) in before:
     NetBSD: patch-ab,v 1.6 1999/11/03 04:18:47 jlam Exp 

diff -x *.orig -urN ./clients/Motif-1.2/mwm/system.mwmrc /usr/pkgsrc/x11/lesstif/work.i386.unpatched/lesstif-0.89.4/clients/Motif-1.2/mwm/system.mwmrc
--- ./clients/Motif-1.2/mwm/system.mwmrc	Wed Jan 13 16:11:05 1999
+++ /usr/pkgsrc/x11/lesstif/work.i386.unpatched/lesstif-0.89.4/clients/Motif-1.2/mwm/system.mwmrc	Wed Nov 24 17:30:51 1999
@@ -4,7 +4,7 @@
 Menu DefaultRootMenu
 {
 	"Root Menu"		f.title
-	"New Window"		f.exec "xterm -e /bin/bash &"
+	"New Window"		f.exec "xterm &"
 	"Shuffle Up"		f.circle_up
 	"Shuffle Down"		f.circle_down
 	"Refresh"		f.refresh
